guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,389 @@
+
+#### Tracking payments
+
+Without using labels, receivers have no way of knowing who may have paid them since they do not interact with the sender to provide them a on-chain address.
+
+{% include tip/recommendation.html %}
+
+Labels in silent payments are critical if receivers such as businesses need to match payments with customers.
+
+{% include tip/close.html %}
+
+Here's how businesses such as exchanges, merchants and vendors can use labels:
+- Invoices: merchant software like [BTCPay](https://btcpayserver.org/) must use labels to match incoming payments with invoices and/or provide static addresses to repeat customers.
+- Bitcoin exchange deposits: Customers looking to sell/deposit bitcoin at exchanges encounter friction when their deposit addresses change constantly. Labelled static addresses for users fix this without address reuse while retaining the ability to match deposits to customers.
+
+
+### Contacts
+
+ Since senders can safely use the same static address for multiple payments, it is natural for them to want to store these for future use. Contacts are a great way to do this in terms that users can intuit: names and faces. The [contacts](/guide/daily-spending-wallet/contacts/) page provides guidance about the topic.
+
+ Silent payments (along with bolt-12) allow applications to center their payments experience around people instead of addresses, something that was not advisable before due to issues with on-chain address reuse.
+
+
+{% include image-gallery.html pages = page.images_contacts %}
+
+{% include tip/tip.html %}
+
+ Contacts are not just for sending. The receiver can create contacts for parties they only receive bitcoin from. When they share a labelled static addresses with others, such as employers or customers, creating a contact can help with invoicing, tracking payments and coin selection.
+
+{% include tip/close.html %}
+
+
+## Setup
+
+Applications can take a variety of approaches to set up and communicate silent payment wallets. Benefits of silent payments such as Contacts and auto-labelling should be explained during setup. Some wallets may use unique or significant locations (mobile widget, custom app logos, watch faces) to place this static address for easy retrieval, and may highlight the same during setup.
+
+{% include image-gallery.html pages = page.images_sp-onboarding %}
+
+
+## Sending
+
+With the improvements to contacts, on-chain send flows may start from a number of [entry points](/guide/daily-spending-wallet/sending/#payment-entry-points). When users start with obtaining a static address, every on-chain address derived from it will be visibly different from the static address the sender started with. This is likely to be confusing for users. Applications should take measures such as short explainers to avoid confusion on the users part. Test transactions are another good way to help users deal with this.
```

Comment:
```suggestion
With the improvements to contacts, on-chain send flows may start from a number of [entry points](/guide/daily-spending-wallet/sending/#payment-entry-points). When users start with obtaining a static address, every on-chain address derived from it will be visibly different from the static address the sender started with. This is likely to be confusing for users. Applications should take measures such as short explainers to avoid confusion on the user's part. Test transactions are another good way to help users deal with this.
```